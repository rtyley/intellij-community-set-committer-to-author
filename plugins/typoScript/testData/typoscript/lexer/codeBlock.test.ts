myObject {
field = title
   ifEmpty {  IGNORED_TEXT
       data = leveltitle:0  } bracket is ignored
     [i = 5]
    }    some IGNORED_TEXT
}
} this brace is also ignored by parser

second.object{
{ ignored opening bracket
 .10 = left
 prop < test
 text(
  } ignored bracket
 )
}

third.object{
  do.not.ignore{
    i=5
  }
}