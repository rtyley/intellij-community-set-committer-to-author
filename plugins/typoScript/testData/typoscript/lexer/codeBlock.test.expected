object path entity ('myObject')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('field')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
assignment value ('title')
WHITE_SPACE_WITH_NEW_LINE ('
   ')
object path entity ('ifEmpty')
WHITE_SPACE (' ')
{ ('{')
IGNORED_TEXT ('  IGNORED_TEXT')
WHITE_SPACE_WITH_NEW_LINE ('
       ')
object path entity ('data')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
assignment value ('leveltitle:0  } bracket is ignored')
WHITE_SPACE_WITH_NEW_LINE ('
     ')
CONDITION ('[i = 5]')
WHITE_SPACE_WITH_NEW_LINE ('
    ')
} ('}')
IGNORED_TEXT ('    some IGNORED_TEXT')
WHITE_SPACE_WITH_NEW_LINE ('
')
} ('}')
WHITE_SPACE_WITH_NEW_LINE ('
')
} ('}')
IGNORED_TEXT (' this brace is also ignored by parser')
WHITE_SPACE_WITH_NEW_LINE ('

')
object path entity ('second')
. ('.')
object path entity ('object')
{ ('{')
WHITE_SPACE_WITH_NEW_LINE ('
')
IGNORED_TEXT ('{ ignored opening bracket')
WHITE_SPACE_WITH_NEW_LINE ('
 ')
. ('.')
object path entity ('10')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
assignment value ('left')
WHITE_SPACE_WITH_NEW_LINE ('
 ')
object path entity ('prop')
WHITE_SPACE (' ')
< ('<')
WHITE_SPACE (' ')
object path entity ('test')
WHITE_SPACE_WITH_NEW_LINE ('
 ')
object path entity ('text')
( ('(')
WHITE_SPACE_WITH_NEW_LINE ('
  ')
multiline value ('} ignored bracket')
WHITE_SPACE ('
 ')
) (')')
WHITE_SPACE_WITH_NEW_LINE ('
')
} ('}')
WHITE_SPACE_WITH_NEW_LINE ('

')
object path entity ('third')
. ('.')
object path entity ('object')
{ ('{')
WHITE_SPACE_WITH_NEW_LINE ('
  ')
object path entity ('do')
. ('.')
object path entity ('not')
. ('.')
object path entity ('ignore')
{ ('{')
WHITE_SPACE_WITH_NEW_LINE ('
    ')
object path entity ('i')
= ('=')
assignment value ('5')
WHITE_SPACE_WITH_NEW_LINE ('
  ')
} ('}')
WHITE_SPACE_WITH_NEW_LINE ('
')
} ('}')